coverage-tests.js

```diff
@@ -1080,22 +1080,22 @@ test('should send dom and location when check window', {
   page: 'Default',
   variants: {
     '': {vg: false},
-    'with vg': {vg: true, skipEmit: true},
+    'with vg': {vg: true},
   },
   test({driver, eyes, assert, helpers}) {
     eyes.open({appName: 'Applitools Eyes SDK', viewportSize})
     driver.executeScript('window.scrollTo(0, 350)')
-    driver.executeScript('document.documentElement.setAttribute("data-applitools-expected-frame", "true");')
+    driver.executeScript('document.documentElement.setAttribute("data-applitools-target", "true");') // because we don't have dom.getNodesByTagName
```

Comment:
Curious to know why would we need to have `dom.getNodesByTagName`?
If you would try to find `html` element in dom it will be always the first one. But you need to work with the one in the active frame, so probably marking is the only solution.